evm/client/errors.go

```diff
@@ -717,8 +717,12 @@ const (
 )
 
 func IsTooManyResults(err error, clientErrors config.ClientErrors) bool {
-	var rpcErr rpc.Error
+	// Context timeouts often occur when recieving too many results from RPCs
+	if pkgerrors.Is(err, context.DeadlineExceeded) {
```

Comment:
Can we just call the std lib directly?
```suggestion
	if errors.Is(err, context.DeadlineExceeded) {
```
I think these only exist so you could import pkg/errors as just `errors` if you wanted, but we've aliased it here already anyways :shrug: